developer-docs-site/docs/nodes/validator-node/operator/staking-pool-operations.md

```diff
@@ -15,26 +15,24 @@ The current required minimum for staking is 1 million APT.
 Important There is no upgrade mechanism for the staking contract from staking pool to delegation pool. A new delegation pool would have to be created.
 :::
 
-## Connect to Aptos network
 
-[Connect to the Aptos network](./connect-to-aptos-network.md) and start your node with `validator-identity` and `validator-fullnode-identity` addresses using your staking pool address.
+## Initialize a staking pool
 
-## Initializing the stake pool
+1. Create a staking pool by running the below command:
```

Comment:
Before this step, maybe mention that they should already have an owner account exist, with 1M APT to proceed with initializing staking pool